### PR TITLE
Add 8.14 what’s new

### PR DESCRIPTION
**Pull request by bmorelli25**

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -3,24 +3,101 @@
...
+[role="screenshot"]
+image::images/wn-new-synthetics.png[What's new in 8.14: Screenshot of the Edit SLO page]
+
+[float]
+== Elastic AI Assistant for Observability enhancements
+
+[float]
+=== AI Assistant context-based starter prompts
+
+We added starter prompts to help new users with their onboarding on the AI Assistant. Default prompts can be overridden by contextual ones set by the current application. Starter prompts provide hints to the user on how to use the AI Assistant like "Are there any open alerts?" These prompts can be application- or data-specific.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:73]: This paragraph has a few problems:
- We are really overloading the term "onboarding" at Elastic these days. Is this simply saying that the starter prompts help new users learn to use the AI Assistant?
- It's not clear who is doing the action here: "Default prompts can be overridden by contextual ones set by the current application." Is this simply saying that you can override the default prompts? 
- This sentence should probably come after the first sentence where we introduce the concept of starter prompts: "Starter prompts provide hints to the user on how to use the AI Assistant like "Are there any open alerts?" 
- I'm not exactly sure what this sentence means: "These prompts can be application- or data-specific."  I'd elaborate, provide more examples, or just remove this sentence.

**Review by dedemorton — COMMENTED** [review]
A few editorial comments.

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -3,24 +3,101 @@
...
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:73]:
```suggestion
Using the Elastic AI Assistant is even easier in version 8.14 with the addition of starter prompts. Starter prompts provide hints on how to use the AI Assistant. For example, "Are there any open alerts?" Starter prompts are tailored to both the application that you're using and the data you have in Elastic.
```

**Comment by bmorelli25** [comment]
Thanks, @dedemorton !

**Review by dedemorton — APPROVED** [review]
**L**ooks **G**roovy **T**o **M**e

**Pull request merged**